docs/Cookbook/Deploy/idxrc.php

```diff
@@ -19,9 +24,9 @@
 
 return \Idephix\Config::fromArray(
     array(
-        \Idephix\Config::ENVS => $environments,
-        \Idephix\Config::SSHCLIENT => new \Idephix\SSH\SshClient(new \Idephix\SSH\CLISshProxy()),
-        \Idephix\Config::EXTENSIONS => array(
+        'envs' => $environments,
+        'ssh_client' => new \Idephix\SSH\SshClient(new \Idephix\SSH\CLISshProxy()),
```

Comment:
Yes, it is optional indeed: SshClient will create a CliSshProxy instance in null is passed